include/RAJA/policy/hip/policy.hpp

```diff
@@ -94,6 +94,15 @@ struct hip_work : public RAJA::make_policy_pattern_launch_platform_t<
                        RAJA::Platform::hip> {
 };
 
+#if defined(RAJA_ENABLE_HIP_INDIRECT_FUNCTION_CALL)
+struct unordered_hip_loop_y_block_iter_x_threadblock_average
```

Comment:
Wow! That is some policy name.

Reply:
I tried to make it as descriptive as possible. Its a copy of the cuda version.

Reply:
Its trying to say that loops or for_alls are mapped onto blocks in the y dimension and iterations in the loops are mapped to threadblocks in the x dimension.

Reply:
got it.

Reply:
> Its trying to say that loops or for_alls are mapped onto blocks in the y dimension and iterations in the loops are mapped to threadblocks in the x dimension.

That's pretty much what I had in mind when I first read it :grinning: